### PR TITLE
v17

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "manifest_version": 2,
   "name": "__MSG_extensionName__",
   "default_locale": "en_US",
-  "version": "16",
+  "version": "17",
```
Comment (pdehaan) [inline src/manifest.json]: Q: does src/manifest_fenix.json version number need to be in sync, or are they completely separate?

**Comment by sharmaaditya570191** [comment]
Thanks for your response @bakulf . Could you please share your slack invite link where I may communicate with you?

File: src/manifest.json (hunk 1 above)
Comment (bakulf) [reply src/manifest.json]: The version is not used in fenix. We don't update the extension.

**Comment by bakulf** [comment]
> Thanks for your response @bakulf . Could you please share your slack invite link where I may communicate with you?

There is a bit of documentation here: https://wiki.mozilla.org/NDA-Slack

**Pull request merged**